app/src/main/java/com/fstyle/androidtrainning/screen/splash/SplashActivity.java

```diff
@@ -7,6 +7,7 @@
 import android.os.Handler;
 import android.support.annotation.NonNull;
 import android.support.v4.app.ActivityCompat;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports